src/main/java/org/itsallcode/openfasttrace/report/view/AbstractViewContainer.java

```diff
@@ -31,20 +31,49 @@
 public abstract class AbstractViewContainer implements ViewableContainer
 {
     protected List<Viewable> children;
+    protected final String id;
+    protected final String title;
 
     /**
      * Create a new instance of type {@link AbstractViewContainer}.
      */
     public AbstractViewContainer()
     {
-        this.children = new ArrayList<>();
+        this(null, null);
```

Comment:
If id can be null: maybe make it Optional<String>?

Reply:
If it was part of the external API, yes. But `null` does not appear on the outside, so I think this is okay.